extensions/cpsection/backup/view.py

```diff
@@ -224,6 +224,7 @@ def __init__(self, operation, view):
         # check the backend availables, if there are more than one
         # show to the user to select one
         if len(self._view.manager.get_backends()) > 1:
+            message = None
```

Comment:
When _message_ is set to None, then `_ask_options` will fail on the `set_text` call.  What made you choose None?

Reply:
Yeah you are probably right,But if none of the if statement runs , what value will be assigned to message. We must initialize message to some value before if statements.

Reply:
Yes, if you choose to change the code to defend against a vulnerability that a tool advises you about (that _operation_ is not one of the tested values), then you are duty bound to think about the outcomes of that change.  What value and type of _message_ will be accepted?  That's what you have to decide.

As for me, I don't bother making this kind of change, because I know the tool is giving advice in the context that is questionable.  There is no call to instantiate OperationPanel that has an _operation_ value other than the values that are tested for.

Is a more appropriate solution to rename OperationPanel as _OperationPanel so that it is treated as a private class?  Whether this is acceptable depends on the tool you are using; and I don't know what that tool is, nor which version.

Reply:
> Is a more appropriate solution to rename OperationPanel as _OperationPanel so that it is treated as a private class? Whether this is acceptable depends on the tool you are using; and I don't know what that tool is, nor which version.

I am getting any such warning. I am using Pycharm as a tool.

Reply:
Okay, then I don't know why you are changing the code in this patch band.

Reply:
Yeah The warning was probably given by that Pycharm tool and there is not a reason to solve that, I have removed those commits.